<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_3"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>161</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="4">
        <widget class="QComboBox" name="BoxChangeCharts"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="PauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="StartButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="BreakButton">
         <property name="text">
          <string>Break</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2" colspan="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <widget class="QLineEdit" name="lineC">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>80</y>
            <width>60</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineFrom">
          <property name="geometry">
           <rect>
            <x>129</x>
            <y>28</y>
            <width>61</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineTo">
          <property name="geometry">
           <rect>
            <x>129</x>
            <y>54</y>
            <width>61</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineA">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>28</y>
            <width>60</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>54</y>
            <width>15</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>B=</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineB">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>54</y>
            <width>60</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>81</x>
            <y>80</y>
            <width>41</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Step</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>28</y>
            <width>15</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>A=</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>80</y>
            <width>15</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>C=</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>81</x>
            <y>54</y>
            <width>41</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>81</x>
            <y>28</y>
            <width>41</width>
            <height>22</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>From</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineStep">
          <property name="geometry">
           <rect>
            <x>129</x>
            <y>80</y>
            <width>61</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
